_data/rules.yml

```diff
@@ -391,6 +391,32 @@ groups:
                 severity: critical
                 for: 2m
 
+      - name: VMware
+        exporters:
+          - name: pryorda/vmware_exporter
+            doc_url: https://github.com/pryorda/vmware_exporter
+            rules:
+              - name: Virtual Machine Memory Warning
+                description: 'High memory usage on {{ $labels.instance }}: {{ $value | printf "%.2f"}}%'
+                query: '((vmware_vm_mem_usage_average / 100) >= 90) and ((vmware_vm_mem_usage_average / 100) < 95)'
+                severity: warning
+                for: 30m
+              - name: Virtual Machine Memory Critical
+                description: 'High memory usage on {{ $labels.instance }}: {{ $value | printf "%.2f"}}%'
+                query: '((vmware_vm_mem_usage_average / 100) >= 95)'
+                severity: critical
+                for: 5m
+              - name: High Number of Snapshots
+                description: "High snapshots number on {{ $labels.instance }}: {{ $value }}"
+                query: '(vmware_vm_snapshots < 3)'
+                severity: warning
+                for: 30m
+              - name: Very High Number of Snapshots
```

Comment:
It checks the time since the last snapshot

Reply:
That's my bad.  I'm also going to adjust the number down to 7 days according to VMware best practices outlined here, which though a bit aggressive, is documented as best practices.  Will also update accordingly.